### PR TITLE
Multiple if statements changed to else-if in the script

### PR DESCRIPTION
**Pull request by baymac**

### CODE REVIEW

File: static/set_proxy.sh [hunk 1]
```diff
@@ -128,88 +122,41 @@ fi
 echo "Git proxy set"
 }
 
-#setting docker proxy
```
Comment (thealphadollar) [inline static/set_proxy.sh:131]: Why have you removed docker proxy?

**Review by thealphadollar — CHANGES_REQUESTED** [review]

File: static/set_proxy.sh (hunk 1 above)
Comment (baymac) [reply static/set_proxy.sh:131]: While merging with the upstream repository, I didn't address the conflicts, so my repository kept my version of the script(without docker function).

File: static/set_proxy.sh [hunk 2]
```diff
@@ -30,17 +30,11 @@ if hash git 2>/dev/null; then
   git config --global --unset http.proxy
   git config --global --unset https.proxy
 fi
-if hash docker 2>/dev/null; then
```
Comment (thealphadollar) [inline static/set_proxy.sh]: Still not there?

**Review by thealphadollar — CHANGES_REQUESTED** [review]

**Comment by thealphadollar** [comment]
@baymac Please look into the new change I've requested.

File: static/set_proxy.sh (hunk 2 above)
Comment (baymac) [reply static/set_proxy.sh]: I have added that part in the reset_proxy() function.

Comment (baymac) [reply static/set_proxy.sh]: Actually having two versions of a script in the same repository is leading to confusion. I am suggesting we should have one repository for the script and another one for the webapp.

**Comment by thealphadollar** [comment]
Please remove this file from the repo and make changes to the script inside `static` folder.

**Review by thealphadollar — APPROVED** [review]
LGTM! Thank you for the PR.

**Pull request merged**